pkg/azure/aks.go

```diff
@@ -101,6 +101,28 @@ func NewCluster(ctx context.Context, subscriptionId, resourceGroup, name, locati
 	return nil
 }
 
+func PutCluster(ctx context.Context, subscriptionId, resourceGroup string, mc *armcontainerservice.ManagedCluster) error {
```

Comment:
this probably shouldn't be exported. makes the interface between packages not clean because we would provide access to everything

Reply:
fixed